spec/e2e/e2e_helper.rb

```diff
@@ -219,3 +223,17 @@ def bootstrap_flask
   end
   on hosts, 'python3 -m pip install flask'
 end
+
+def dl_ctr_logs
+  dir = 'tmp/e2e_ctr/'.tap { |d| `mkdir -p #{d}` }
```

Comment:
Maybe make this `File.join(proj_root, 'tmp/e2e_ctr')` since relative path might have weird results in CI/CD pipelines.

Reply:
```suggestion
  dir = File.join(proj_root, 'tmp/e2e_ctr').tap { |d| `mkdir -p #{d}` }
```